Casks/f/follow.rb

```diff
@@ -0,0 +1,37 @@
+cask "follow" do
+  version "0.0.1,21"
+  sha256 "eeddce6afcdfa37e1688fb0328b8562a47f81407e4cd6922a7b0ae2591f72460"
+
+  url "https://github.com/RSSNext/Follow/releases/download/v#{version.csv.first}-alpha.#{version.csv.second}/Follow-#{version.csv.first}-alpha.#{version.csv.second}-macos-universal.dmg",
+      verified: "github.com/RSSNext/Follow/"
+  name "Follow"
+  desc "Next generation information browser"
+  homepage "https://follow.is/"
+
+  livecheck do
+    url :url
+    regex(/^v(\d+\.\d+\.\d+)-alpha\.(\d{2})$/i)
+    strategy :github_releases do |json, regex|
+      json.map do |release|
+        next if release["draft"] || release["prerelease"]
+
+        match = release["tag_name"]&.match(regex)
+        next if match.blank?
+
+        "#{match[1]},#{match[2]}"
+      end
+    end
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+(?:[._-]alpha[._-]?\d+)?)$/i)
```

We can simplify the `livecheck` by including ‘alpha’ as part of the version string. Making the suffix optional should also ensure that the `livecheck` continues to work once upstream starts publishing stable versions.

In addition, we are using the default `Git` strategy to match git tags, which prevents the unnecessary API calls that come with using the `GithubReleases` strategy.